client/src/index.css

```diff
@@ -1,13 +1,281 @@
-body {
+/* NAV */
+p{
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif
+}
+#nav{
+  display:flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border-bottom: 4px solid black;
+  margin-left:-10px;
+}
+
+#nav2 {
+  display:flex;
+  flex-direction: row;
+}
+
+#about {
+  margin: 20px;
+}
+
+#about:hover{
+  color:#99ccff;
+  cursor: pointer
+}
+
+#projects {
+  margin: 20px
+}
+
+#projects:hover{
+  color:#99ccff;
+  cursor: pointer
+}
+
+#interests {
+  margin: 20px
+}
+
+#interests:hover{
+  color:#99ccff;
+  cursor: pointer
+}
+
+#work {
+  margin: 20px
+}
+
+#work:hover{
+  color:#99ccff;
+  cursor: pointer
+}
+
+#contact {
+  margin: 20px
+}
+
+#contact:hover{
+  color:#99ccff;
+  cursor: pointer
+}
```

Comment:
To minimize the amount of code, we could write this as
```
#contact, #work, #interests {
put css here
}

#contact:hover, #work:hover, #interests:hover{
put css here
}
```

Or we could just put the same class on all of them and manipulate them this through the class rather then ids